pkg/projects/Microsoft.NETCore.App/project.json

```diff
@@ -39,7 +39,9 @@
     "System.Threading.Tasks.Parallel": "4.0.1",
     "System.Threading.Thread": "4.0.0",
     "System.Threading.ThreadPool": "4.0.10",
-    "Libuv": "1.9.1"
+    "Libuv": "1.9.1",
+    "Microsoft.NETCore.Platforms": "1.0.2-servicing-24703-01",
```

Comment:
I kept them out of order for making it explicit that these were getting updated for servicing.